src/ripple.js

```diff
@@ -78,8 +78,8 @@ var Ripple = {
             // No need to set positioning because ripple should be child of target and to it's relative position.
             // rippleContainer.style.left    = left + (((window.pageXOffset || document.scrollLeft) - (document.clientLeft || 0)) || 0) + "px";
             // rippleContainer.style.top     = top + (((window.pageYOffset || document.scrollTop) - (document.clientTop || 0)) || 0) + "px";
-            rippleContainer.style.width   = width + "px";
-            rippleContainer.style.height  = height + "px";
+            rippleContainer.style.width   = "100%";
```

Comment:
does it work ok when parent element has `border`?

Reply:
Good catch. I added it to this width calculation. There will be a problem if element will have different border sizes, but original calculation also does not take this into account:
```
var targetBorder = parseInt((getComputedStyle(target).borderWidth).replace('px', ''));
```